index.js

```diff
@@ -209,6 +209,25 @@ const template = (data, name) => {
 `.trim();
 };
 
+const fileName = (data, name) => {
+  const createdDate = new Date(data.created_at);
+  const month = createdDate.getMonth() + 1;
+  const day = createdDate.getDate();
+
+  const datePart = [
+      createdDate.getFullYear(),
+      (month > 9 ? '' : '0') + month,
```

Comment:
Use a function like `function pad(n){return n<10 ? '0'+n : n}`to reduce the duplicate code for leading zeros

Reply:
So full code:
``` 
const fileName = (data, name) => {
  const createdDate = new Date(data.created_at);
  const month = createdDate.getMonth() + 1;
  const day = createdDate.getDate();

  function pad(n){return n<10 ? '0'+n : n}
  const datePart = [
      createdDate.getFullYear(),
      pad(month),
      pad(day)
  ].join('');

  const timePart = [
     pad(createdDate.getHours()),
     pad(createdDate.getMinutes()),
  ].join('');

  return [datePart, timePart, name].join('_');
};
```